restart_from_hook.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-echo "Restarting from hook"
```

Comment:
Why did you delete this file?

Reply:
Becasue it wasn't used anywhere!

Reply:
Please double-check with the production instance of MetaLookup - as far as I know that machine uses e.g. metadata-picker yml and the restart_from_hook.sh.